recipe/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "imbalanced-ensemble" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - path: requirements.txt
```

Comment:
```suggestion
  # we provide this file from our recipe folder because the upstream does not provide one,
  # instead of patching the tarball
  - path: requirements.txt
```